docs/SetupAndTeardown.md

```diff
@@ -0,0 +1,118 @@
+---
+id: setup-teardown
+title: Setup and Teardown
+layout: docs
+category: Introduction
+permalink: docs/setup-teardown.html
+next: mocking
+---
+
+Often while writing tests you have some setup work that needs to happen before tests run, and you have some finishing work that needs to happen after tests run. Jest provides helper functions to handle this.
+
+### Repeating Setup For Many Tests
+
+If you have some work you need to do repeatedly for many tests, you can use `beforeEach` and `afterEach`.
+
+For example, let's say that several tests that interact with a database of cities. You have a method `initializeCityDatabase()` that must be called before each of these tests, and a method `clearCityDatabase()` that must be called after each of these tests. You can do this with:
```

Comment:
This sentence is missing a few words: "For example, let's say that several tests that interact with a database of cities."